### ISSUE TITLE
Not an issue but I would like to help and built dot develop in other platforms (Debian Linux/ FreeBSD)

### ISSUE THREAD

**Issue by gnubsdguy** [dotdevelop/dotdevelop#141]
Hi Hello

I have seen your project, and I was looking for something like this in the following months (I have seen what happened with monodevelop and to be sincere I liked so much that I used every day to learn C#). I began to learn C# and C++ in the last month, I must be sincere that I am just beginner but in any case I have some ideas and I know how to compile programs in Debian Linux and FreeBSD. I know there is so much stuff to learn, but I see this an opportunity to learn something new and to have fun with it. Is there anyway how to contribute and help with the project ? (I have in mind something we can add in the future if this could be accepted).

**Comment by vanackerglenn** [comment]
@gnubsdguy
Sure, have you managed to build the project? 
If you have, you should be able to open dotdevelop, and then open the code and start debugging.
have a look at our roadmap and docs for how to contribute.
https://github.com/dotdevelop/dotdevelop/wiki/DotDevelop-Road-Map

The best way to start is to create your own fork and poke around the codebase imo.

**Comment by gnubsdguy** [comment]
Hi There

Thanks to answer !, well I haven't tried to build the project myself, but I can give a try to build in Debian Linux/FreeBSD and could be this helpful ? I just want to this to get deeply knowledge on C# and .Net develop on Linux/BSD platform.
Anyway, do you accept any comments or suggestion ? I think we can add something else to the project near the future.

**Comment by vanackerglenn** [comment]
@gnubsdguy 
It would help if you were able to build the project yes, because you can use that to help with development.
just make sure you have all dependencies installed, and follow the build steps, it should be fine, I built it earlier today.
If you have suggestions for the roadmap, you can add them here: 
https://github.com/dotdevelop/dotdevelop/issues/109
We're still looking at making a discussion board for those things

**Comment by vanackerglenn** [comment]
@gnubsdguy If you have any issues with the build or with C# and .Net in general, feel free to reach out and ask

**Comment by gnubsdguy** [comment]
Thanks, I will, I haven't done anything yet. Anyway, do you accept
suggestions ?

Thanks so much for your help!

**Comment by vanackerglenn** [comment]
yes, but for the suggestions we have the roadmap thread, it's probably better to leave comments there:
https://github.com/dotdevelop/dotdevelop/issues/109
